canoto.go

```diff
@@ -726,7 +726,7 @@ func FieldTypeFromField[T Field](
 		fieldType = reflect.TypeOf(field).Elem()
 	)
 	if index := slices.Index(types, fieldType); index >= 0 {
-		f.TypeRecursive = uint64(len(types) - index)
+		f.TypeRecursive = uint64(len(types) - index) //nolint:gosec // disable G115
```

Comment:
Rather than disabling `gosec` entirely, we can disable just the single rule:
```suggestion
		f.TypeRecursive = uint64(len(types) - index) //#nosec G115 // False positive
```